doc/source/installation.rst

```diff
@@ -1,3 +1,10 @@
+===========
```

Comment:
https://github.com/Tendrl/ceph_bridge/pull/15

Make this install file as the above PR, move the sample job creation parts to some other doc. not here